app/src/main/java/com/cmput301f20t21/bookfriends/ui/profile/ProfileViewModel.java

```diff
@@ -75,4 +84,33 @@ public void getUserByUid(String uid, OnSuccessCallbackWithMessage<User> onSucces
             onSuccess.run(u);
         });
     }
+
+    /*public void updateCurrentUserEmail(String username, String TAG){
+
+        //update email authentication
+        FirebaseUser user = FirebaseAuth.getInstance().getCurrentUser();
+        if (user != null) {
+            user.updateEmail(username)
+                    .addOnCompleteListener(new OnCompleteListener<Void>() {
+                        @Override
+                        public void onComplete(@NonNull Task<Void> task) {
+                            if (task.isSuccessful()) {
+                                Log.d(TAG, "User email address updated.");
+                            }
+                        }
+                    });
+
+        }
+    }*/
+    public void updateFirestoreUser(String inputEmail, String inputPhone){
+        User firebaseUser = AuthService.getInstance().getCurrentUser();
+        if (firebaseUser != null) {
+            String userId = firebaseUser.getUid();
+            FirebaseFirestore.getInstance().collection("users").document(userId)
```

Comment:
similar to the above comment regarding email, this should be in UserService class, and have a OnCompleteHandler attached to that method instead